scripts/Configure-Environment.ps1

```diff
@@ -74,15 +70,3 @@ Write-Host "Using the following GalleryBaseUrl: $GalleryBaseUrl"
 $env:GalleryBaseUrl = $GalleryBaseUrl
```

Comment:
Either here or tracked by a backlog item -- we should eliminate the PowerShell script completely and fetch the staging base URL in C# as you have done with search.